osgbuild/promoter.py

```diff
@@ -149,16 +177,23 @@ def split_repo_dver(build, known_repos=None):
 
 
 def _parse_list_str(list_str):
-    # split string on spaces or commas
-    items = re.split(r'[ ,]', list_str)
+    # split string on whitespace or commas
+    items = re.split(r'[ ,\t\n]', list_str)
     # remove empty strings from the list
     filtered_items = [_f for _f in items if _f]
     return filtered_items
 
-multiroutes=dict()
-def load_routes(inifile):
-    """Load dict of routes (key is the route name, value is a Route object
-    from a .ini file.
+
+def _commajoin(l):
+    return ", ".join([str(x) for x in sorted(l)])
```

Comment:
Not a big deal but note that the `[` `]` brackets are superfluous here.

Reply:
If you leave it off it will make a generator, right? That's OK too I suppose.

Reply:
Also not a big deal, but the name `_commajoin` does not convey the (presumably) point that the input list will be sorted.

Feel free to ignore but i just had the thought:

```
def join_sorted(sep, seq):
    return sep.join(map(str, sorted(seq)))
```

Reply:
>you leave it off it will make a generator, right?

Yeah it passes in a generator expression in that case, which will work for any function that takes a single sequence argument.  It works either way of course, i just mentioned it in passing.